Add NPC Test Plans (#102)
Gamify-IT/docs: dev-manuals/test-plans/overworld-frontend/plans/npc-load-dialogue.md

> # Name: NPC load dialogue

Reviewer: ```suggestion
# NPC load dialogue
```
Also, what about using the usecase number in the title as well? That'll be better for the `ReadTheDocs` integration:
```suggestion
# NPC load dialogue (`U.NPC.2`)
```

> Number: U.01.2

Reviewer: Didn't we say to make the number the test case identifier?
So that i.e. this testplans number would be `U.NPC.2`?

Version: V0.1, 2022-08-02, create test plan
Author: Florian Wüst, Gilian Rehm
Tester: -

## Description

> This test case verifies that the dialogue lines of a NPC can be loaded dynamically from the data base, and the NPC uses those dialogue lines, if talked to.

Reviewer: ```suggestion
Verifies that the dialogue lines of a NPC can be loaded dynamically from the database, and the NPC uses those dialogue lines, if talked to.
```

## Precondition

> The NPC is configurated in the overworld and corresponding data exists in the backend.

Reviewer: ```suggestion
The NPC is configured in the overworld and corresponding data exists in the backend.
```

## Postcondition

> The dialogue lines stored in the backend are set as the dialogue lines from the NPC.

Reviewer: ```suggestion
The dialogue lines stored in the backend are set as the dialogue lines of the NPC.
```

## Typical procedure

> 1. The tester starts the game with a running data base

Reviewer: ```suggestion
1. The tester starts the game with a running backend
```

> 2. The tester goes to the NPC.
> 3. The tester starts a dialogue.
> 4. The correct lines are shown.

Reviewer: ```suggestion
2. The tester goes to the NPC
3. The tester starts a dialogue
4. The correct lines are shown
```

## Alternative procedures

> 4.1 Not the correct lines are shown.

Reviewer: ```suggestion
4.1 Not the correct lines are shown
```

## Criticality

Medium

## Linkages

> -

Reviewer: `- NPC.3`
or `- U.NPC.3`
